mailman/submit.py

```diff
@@ -61,15 +63,29 @@ def set_status(status):
         sub.save()
         return 0
 
+    # buffer defaults to 7 if value is unset or invalid
+    env_mins = os.getenv('DEADLINE_BUFFER_MINUTES', 7)
```

Comment:
I'd say define this as an ALL_CAPS variable (DEADLINE_BUFFER_MINUTES) under the imports but before the first function